SB2018/2018-02-07.md

```diff
@@ -22,7 +22,7 @@ In this release, we've introduced 2 new coordinate systems, and depricated an ex
 
 As many have noted the [`cartesian`](https://docs.sourcebots.co.uk/api/vision/coordinates/#cartesian-coordinates) system was not implemented, and would raise an exception whenever accessed. This has now been fixed.
  
-The `polar` coordinate system has been replaced with [`spherical`](https://docs.sourcebots.co.uk/api/vision/coordinates/#spherical-coordinates). The new system is more accurate, and better documented than polar, and we recommend updating to it as soon as possible! The available attributes are identical in `polar`, and so will likely be a simple drop-in replacement. Be sure to consult the documentation on exactly now to use the new system. Attempting to use `polar` will still work as it does now, however displays a warning message.
+The `polar` coordinate system has been replaced with [`spherical`](https://docs.sourcebots.co.uk/api/vision/coordinates/#spherical-coordinates). The new system is easier to understand, and better documented than polar. We recommend updating as soon as possible! The available attributes are identical in `polar`, and so will likely be a simple drop-in replacement. Be sure to consult the documentation on exactly now to use the new system. Attempting to use `polar` will still work as it does now, however displays a warning message.
```

Comment:
> The available attributes are identical in `polar`, and so will likely be a simple drop-in replacement.

This is false (by design) -- the new properties spell out the name of the angle units in full (`rot_x_degrees` rather than `rot_x_deg`); sorry I missed this earlier.

I'd originally wanted all the properties to be different (including `distance_metres`, though couldn't think of a good alternative there), so that it would be clear which version of the API you were using. I felt this was important as the angles have _very_ different interpretations.